bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrDetailsCollector.kt

```diff
@@ -51,7 +51,7 @@ internal class AnrDetailsCollector {
         }
     }
 
-    internal fun collectAnrErrorDetails(client: Client, error: Error) {
+    fun collectAnrErrorDetails(client: Client, error: Error) {
```

Comment:
No. It was originally needed as this was called from Java code, and [Kotlin mangles non-public methods](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#visibility) to prevent access from Java. However, this method is effectively not accessible to the public anyway due to the class being `internal`.

Reply:
Uh, I think you changed the access policy on the wrong method?

Reply:
I've updated both methods as neither need to be public.